tensorflow-core/tensorflow-core-api/src/test/java/org/tensorflow/SavedModelBundleTest.java

```diff
@@ -85,88 +85,18 @@ public void loadNonExistentBundle() {
 
   @Test
   public void loader() {
-    try (SavedModelBundle bundle = SavedModelBundle.loader(SAVED_MODEL_PATH)
-        .withTags("serve")
-        .withConfigProto(sillyConfigProto())
-        .withRunOptions(sillyRunOptions())
-        .load()) {
+    try (SavedModelBundle bundle =
+        SavedModelBundle.loader(SAVED_MODEL_PATH)
+            .withTags("serve")
+            .withConfigProto(sillyConfigProto())
+            .withRunOptions(sillyRunOptions())
+            .load()) {
       assertNotNull(bundle.session());
       assertNotNull(bundle.graph());
       assertNotNull(bundle.metaGraphDef());
     }
   }
 
-  @Test
-  public void exportFunctionWithVariables() throws IOException {
```

Comment:
@rnett , is this test being replaced somewhere in this PR? If not, where do we guarantee that we did not break the model export?

Reply:
Oh yeah, I just need to change it over to use `SessionFunction`.